### PR TITLE
[Merged by Bors] - PoST: Remove K3 Pow

### PR DESCRIPTION
**Pull request by fasmat**
## Motivation
This removes K3 Pow from the Post Proof

## Changes
This integrates https://github.com/spacemeshos/post/pull/145

Merge after
- [x] https://github.com/spacemeshos/go-spacemesh/pull/4394
- [x] https://github.com/spacemeshos/post/pull/145

have been merged

## TODO

- [ ] Explain motivation or link existing issue(s)
- [ ] Test changes and document test plan
- [ ] Update documentation as needed

## DevOps Notes

- [x] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [x] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

### CODE REVIEW

**Comment by fasmat** [comment]
bors try

**Comment by fasmat** [comment]
bors try

**Comment by fasmat** [comment]
bors try

**Comment by fasmat** [comment]
bors try

**Comment by fasmat** [comment]
bors try

File: activation/post.go [hunk 1]
```diff
@@ -28,9 +28,8 @@ type PostConfig struct {
 	K1            uint32 `mapstructure:"post-k1"`
 	K2            uint32 `mapstructure:"post-k2"`
 	K3            uint32 `mapstructure:"post-k3"`
-	// Difficulties for K2 and K3 Proofs of Work
+	// Difficulty for K2 Proofs of Work
```
Comment (poszu) [inline activation/post.go]: Now that I looked at this comment I think it's redundant. Wdyt?
```suggestion
```

**Review by poszu — COMMENTED** [review]
This line should be removed too: https://github.com/spacemeshos/go-spacemesh/blob/b2357d4f18619b497aad86308e12c335b0572853/systest/parameters/bignet/smesher.json#L41

**Review by dshulyak — APPROVED** [review]

**Comment by poszu** [comment]
bors merge

**Pull request closed without merging**